x-pack/plugins/ml/public/application/components/custom_urls/custom_url_editor/custom_time_range_picker.tsx

```diff
@@ -73,8 +77,18 @@ export const CustomTimeRangePicker: FC<CustomUrlTimeRangePickerProps> = ({
 
   return (
     <>
-      <EuiSpacer size="s" />
-      <EuiFlexGroup gutterSize={'none'}>
+      <EuiFlexGroup gutterSize={'none'} alignItems="center">
+        <EuiFlexItem grow={false}>
+          <EuiSwitch
+            disabled={disabled}
+            label={i18n.translate('xpack.ml.customUrlsEditor.addCustomTimeRangeSwitchLabel', {
+              defaultMessage: 'Add custom time range?',
```

Comment:
This should be fine without the question mark I guess.

Reply:
Removed in 124d047b2f4b880c65e2cec0bbd980421b74b4bd